src/components/Menu/Menu.js

```diff
@@ -4,19 +4,18 @@ import Button from 'antd/lib/button'
 import { connect } from 'react-redux'
 import { onChangeMenuState } from '../../actions/layout'
 import List from 'antd/lib/list'
-import { getMenuState } from '../../store/selectors';
+import { getMenuState } from '../../store/selectors'
+import { Menu as AntMenu } from 'antd'
+import { CloseOutlined } from '@ant-design/icons'
 
 class Menu extends Component {
-  onChangeMenuState = (nItem) => {
+  onChangeMenuState = nItem => {
     this.props.onChangeMenuState(nItem)
   }
 
   render() {
-    const { 
-      sidebarDocked,
-      menuOpen,
-    } = this.props
-
+    const { sidebarDocked, menuOpen } = this.props
+    console.log(this.props.location)
```

Comment:
Getting an undefined message from this on expanding the menu. Maybe this is then leading to:

![Screenshot 2020-05-12 at 22 42 46](https://user-images.githubusercontent.com/47497682/81749821-41572900-94a4-11ea-871b-c5a6750ec529.png)

and this error:

![Screenshot 2020-05-12 at 22 59 55](https://user-images.githubusercontent.com/47497682/81749849-5338cc00-94a4-11ea-860f-5153b0bf21f0.png)